modules/sdk-core/src/bitgo/tss/eddsa/eddsa.ts

```diff
@@ -200,8 +198,7 @@ export async function offerUserToBitgoRShare(
   const signatureShare: SignatureShareRecord = {
     from: SignatureShareType.USER,
     to: SignatureShareType.BITGO,
-    // TODO(BG-61037): Fix signing with VSS
-    share: /* rShare.v + */ rShare.r + rShare.R,
+    share: rShare.r + rShare.R + rShare.v,
```

Comment:
Why are we putting them all in one string?

Reply:
this the pattern how tss exchanges shares with WP <> sdk. shares are contacted before sending to wp

Reply:
would hope we had delimiter instead of counting bytes